packages/frontend/src/pages/Profile/PartiesOverviewPage/NotificationSettings.tsx

```diff
@@ -43,17 +44,9 @@ export const NotificationSetting = ({ notificationSetting }: NotificationSetting
         value={value}
         badge={badge as BadgeProps}
         linkIcon
-        onClick={() => setShowModal((prev) => !prev)}
+        onClick={() => setNotificationParty(notificationSetting as NotificationAccountsType)}
```

Comment:
Why the cast? The `NotificationSettingsResponse` doesn't look like `NotificationAccountsType`?